cli/hzn.go

```diff
@@ -3,11 +3,14 @@ package main
 
 import (
 	"flag"
-	"github.com/open-horizon/anax/cli/sdo"
-	"github.com/open-horizon/anax/version"
 	"os"
 	"strings"
 
+	"github.com/open-horizon/anax/cli/sdo"
```

Comment:
Did you move these lines around or did the formatting tool do it?

Reply:
it is done by formatting tool